### ISSUE TITLE
Expected scale of iBOT patch loss during pretraining?

### ISSUE THREAD

**Issue by AaronC-BME** [valeoai/Franca#7]
Hi and thanks for releasing Franca — I’ve been experimenting with the pretraining and had a quick question regarding the patch loss.

When running the iBOT patch loss, I noticed that the ibot loss starts relatively high (in the thousands), while the rest of the loss value seems relatively normal. 

Is there a typical or expected range for the iBOT patch loss during training? For example, is it normal to see values in the range of 1–100, or can it go much higher depending on data/model configuration?

Just trying to sanity-check whether my setup is behaving as expected.

**Comment by shashankvkt** [comment]
Hi @AaronC-BME 
Thanks a lot for your interest in Franca!

It looks like there might be an issue with your setup. I recommend checking out the [Franca-B-logfile](https://raw.githubusercontent.com/valeoai/Franca/refs/heads/main/logfiles/In21K/Franca_B.txt) to compare the expected range of losses.

Just as a note—Franca uses 5 Matryoshka nestings, so the total training loss starts around 60. You can use that as a reference point and scale it accordingly based on the number of nestings you're using.

Let us know if you need any further help.
